docs/docs/fundamentals/troubleshooting.md

```diff
@@ -28,4 +28,37 @@ Depending on how spread is used you may try one of the following alternatives:
 ### `Export namespace should be first transformed by 'babel/plugin-proposal-export-namespace-from'`
 
 This error usually happens, when you forget to add the babel plugin in your `babel.plugin.js`. Please make
-sure you have added it.
\ No newline at end of file
+sure you have added it.
+
+### Multpile versions of Reanimated was detected
+
+This error usually happens when in your project exists more than one instance of Reanimated. It can occur when some of your dependency has installed Reanimated inside their own `node_modules` instead of using it as a peer dependency. In this situation cause that two different versions of Reanimated JS module try to use the same Native Module. You can resolve this problem manually by modifying your `package.json` file.
+
+You can check which libraries are using Reanimated, for example, with the command:
+```bash
+npm why react-native-reanimated
+``` 
```

Comment:
```suggestion
* `yarn why react-native-reanimated`
* `npm why react-native-reanimated`
```

Reply:
I think that `npm why` is more informative than `yarn why`:
<img width="521" alt="image" src="https://user-images.githubusercontent.com/36106620/183056014-2925ed43-424e-4989-9c9c-b383265cdae4.png">